src/main/java/edu/kis/vh/nursery/defaultCountingOutRhymer.java

```diff
@@ -2,33 +2,38 @@
 
 public class defaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	private static final int INT = -1;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.